### PR TITLE
[DOM-133] Obtain target device tokens to send message

### PR DESCRIPTION
**Pull request by ms-darianbenito**
This PR is to obtain target device tokens to send a message. 
image

Related to [DOM-133](https://makingsense.atlassian.net/browse/DOM-133)

### CODE REVIEW

File: Doppler.PushContact/Services/PushContactService.cs [hunk 1]
```diff
@@ -231,6 +231,40 @@ public async Task AddHistoryEventsAsync(IEnumerable<PushContactHistoryEvent> pus
...
+            try
+            {
+                var pushContactsFiltered = await (await PushContacts.FindAsync(filter, options)).ToListAsync();
+
+                return pushContactsFiltered
+                    .Select(x => x.GetValue(PushContactDocumentProps.DeviceTokenPropName).AsString);
+            }
+            catch (Exception ex)
+            {
+                _logger.LogError(ex, $"Error getting {nameof(PushContactModel)}s by {nameof(domain)} {domain}");
```
Comment (andresmoschini) [inline Doppler.PushContact/Services/PushContactService.cs:262]: @elsupergomez always recommends structured logging

```suggestion
                _logger.LogError(ex, "Error getting models by domain {domain}", domain);
```

Comment (ms-darianbenito) [reply Doppler.PushContact/Services/PushContactService.cs:262]: I am using string interpolation in all the solution but I will keep it in mind for a future refactor. Thanks!

**Comment by hrocca** [comment]
🚀

File: Doppler.PushContact/Services/PushContactService.cs (hunk 1 above)
Comment (ms-darianbenito) [reply Doppler.PushContact/Services/PushContactService.cs:262]: Agregué una tarea para que no se pierda esta sugerencia: https://makingsense.atlassian.net/browse/DOM-137. 
CC @hrocca

**Pull request merged**